src/httpserver/handlers/parameter_decode.clj

```diff
@@ -0,0 +1,12 @@
+(ns httpserver.handlers.parameter-decode)
+
+(defn- decode [parameters]
+  (java.net.URLDecoder/decode parameters))
+
+(defn- build-body [parameters]
+  (decode (clojure.string/replace parameters #"&" "\n"))
```

Comment:
1. I would prefer you add `clojure.string` with a require statement in the `ns` declaration—that way it's easy to know at a quick glance what external namespaces/libraries/etc. are being called in the file.

2. I think you can do the same for `java.net.URLDecoder` in the function above this.

3. nit: closing paren on the line below this should be up (same with the `call` function below)